### PR TITLE
DRILL-5803: Show the hostname for each minor fragment in operator table

### PR DESCRIPTION
**Pull request by kkhatua**
Based on DRILL-4909, list hostnames for each minor fragment within an operator table to identify any problematic host

### CODE REVIEW

**Comment by kkhatua** [comment]
@paul-rogers Can you please review this minor UI update?

File: exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java [hunk 1]
```diff
@@ -75,24 +75,31 @@ public ProfileWrapper(final QueryProfile profile) {
     final List<OperatorWrapper> ows = new ArrayList<>();
     // temporary map to store (major_id, operator_id) -> [(op_profile, minor_id)]
     final Map<ImmutablePair<Integer, Integer>, List<ImmutablePair<OperatorProfile, Integer>>> opmap = new HashMap<>();
+    // temporary map to store (major_id, operator_id) -> [minorFragHostname]
```
Comment (paul-rogers) [inline exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java]: Effectively, what this is doing is converting the opmap from a map of immutable pairs to a map of immutable triples. About here, we might want to restructure the map to point to a class which holds the two elements in the first pair, plus the host name.

Why? Simpler. Easier to keep class fields in sync than to keep two separate lists in sync.

Comment (kkhatua) [reply exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java]: +1 on this comment. However, I didn't what to make the PR look to radically alter what is already there. I'll incorporate the change. This should go nicely with a fix for DRILL-5802 and make life a bit easier

**Comment by kkhatua** [comment]
@paul-rogers  Made the changes

**Comment by paul-rogers** [comment]
The fixes are an improvement. In some future commit, an actual class would be better than nested immutable pairs. At present, the code has become a bit cryptic and Python-like. But, we can do that improvement later.
+1

**Pull request closed without merging**